challenges/backend/pom.xml

```diff
@@ -0,0 +1,77 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Overall, I have found that while Gradle is powerful for handling building and dependencies, Maven is much simpler to understand and has broader support in development tools. Most development teams I have worked with in the past are much more familiar with Maven, and since Maven building and dependency management is very simple to set up for small projects such as this REST API and Database converter, I chose to go with Maven, at least for the initial setup of the projects. The use case for Gradle becomes more obvious when working with a larger project that needs more customization in building. For large scale projects like these Maven can be very difficult to work with in situations where Gradle would likely perform much better due to its customizable scripting.